src/python/WMCore/ReqMgr/Service/Request.py

```diff
@@ -438,6 +445,19 @@ def _handleNoStatusUpdate(self, workload, request_args, dn):
                 msg = "There were unhandled arguments left for no-status update."
                 raise InvalidSpecParameterValue(msg) from ex
 
+        # make call to MSTransferor
+        try:
+            # http POST request to /ms-transferor/data with {'workflow': <workflow>}
+            if self.msTransferorUrl:
+                makeHttpRequest(self.msTransferorUrl, {'workflow': workload.name()}, method='POST')
+                cherrypy.log('Updated workflow name "{}" in MSTransferor'.format(workload.name()))
+            else:
+                cherrypy.log("WARNING: unable to make HTTP request to MSTransferor service, no ms_transferor_url configuration found")
+        except Exception as ex:
+            msg = "ERROR: fail to update MSTransferor"
+            cherrypy.log(msg + " Reason: %s" % ex)
+            raise cherrypy.HTTPError(404, msg)
```

Comment:
I think it would be dangerous to convert eventual generic exception to `http` error 404. One alternative, if we want guaranties we are about to hit an exception only due to the request made to MSTransferror,  would be to not raise the high level python builtin `RuntimeError` exception at `makeHttpRequest`  (from which several others derive, such as: NotImplementedError , PythonFinalizationError, RecursionError), but instead use a more specific one. Look at my comment bellow.

Reply:
it is always debatable should we use 404 or 500 HTTP error if server experience a problem. I'll leave it to Alan to decide.

Reply:
ok

Reply:
changed to 500